data/brands/shop/hardware.json

```
@@ -309,19 +309,7 @@
     {
       "displayName": "Würth",
       "id": "wurth-d9914d",
-      "locationSet": {
-        "include": [
-          "be",
-          "cz",
-          "de",
-          "fi",
-          "fr",
-          "it",
-          "no",
-          "sk"
-        ]
-      },
-      "note": "https://github.com/osmlab/name-suggestion-index/issues/5500",
+      "locationSet": {"include": ['001'], "exclude": ['us', 'ca', 'jp', 'ru', 'tw', 'cn']},
```

Strings in json require double quotes

```suggestion
      "locationSet": {"include": ["001"], "exclude": ["us", "ca", "jp", "ru", "tw", "cn"']},
```

---

yes, my tests on https://location-conflation.com/?referrer=nsi misled me, it went well with simple quillemets